lib/tasks/version.rake

```diff
@@ -36,4 +36,31 @@ namespace :version do
       File.write(spec_file, spec)
     end
   end
+
+  desc "Bump package version and create a simple class for acessing the version from sources"
+  task wbump: :bump do
+    puts "Generating helper class"
+
+    Dir.glob("**/lib/*") do |filename|
+      next if !File.directory?(filename)
+
+      module_name = File.basename(filename).downcase
+      target_file = "#{module_name}_version.rb"
```

Comment:
The usual convention for a version file is to use `version.rb`, see `find /usr/lib64/ruby/gems/ -name "*version*.rb"`.